packages/request-node/src/request/confirmedTransactionStore.ts

```diff
@@ -1,50 +1,52 @@
-import { DataAccessTypes } from '@requestnetwork/types';
-import Keyv, { Store } from 'keyv';
+import { DataAccessTypes, StorageTypes } from '@requestnetwork/types';
+import { SubgraphClient } from '@requestnetwork/thegraph-data-access';
 
 /**
  * Class for storing confirmed transactions information
  * When 'confirmed' event is received from a 'persistTransaction', the event data are stored.
  * The client can call the getConfirmed entry point, to get the confirmed event.
  */
 export default class ConfirmedTransactionStore {
```

Comment:
The header comment could use a refresh.

I considered requesting a class name change, but in the end decided it was okay. 

```suggestion
/**
 * Class for storing confirmed transaction information
 * When 'confirmed' event is received from a 'persistTransaction', the event data is
 * stored and indexed by the storage subgraph. The client can call the 
 * getConfirmedTransaction endpoint, to get the confirmed event.
 */
export default class ConfirmedTransactionStore {
```

Reply:
thanks 👍